browser/ui/views/profiles/brave_profile_menu_view.cc

```diff
@@ -25,7 +25,7 @@
 
 namespace {
 bool IsGuest(Profile* profile) {
```

Comment:
mb we don't need this whole function now?

Reply:
Correct, fixed in https://github.com/brave/brave-core/commit/5adfdcb18e7949701c005d18070216a13016aac4